1-js/02-first-steps/02-structure/article.md

```diff
@@ -1,159 +1,159 @@
-# Code structure
+# কোডের গঠন
 
-The first thing we'll study is the building blocks of code.
+প্রথমেই আমরা শিখে নিব কোড লিখতে কি কি লাগে।
 
-## Statements
+## স্টেটমেন্ট
 
-Statements are syntax constructs and commands that perform actions.
+স্টেটমেন্ট হল কাজ করার জন্য নির্দেশনা দেয়ার উপায়।
 
-We've already seen a statement, `alert('Hello, world!')`, which shows the message "Hello, world!".
+আমরা ইতিমধ্যেই একটি স্টেটমেন্ট দেখেছি, `alert('হ্যালো, ওয়ার্ল্ড!')`, যা এই ম্যাসেজটি দেখায় - "হ্যালো, ওয়ার্ল্ড!".
```

Comment:
কমেন্ট ব্যাতীত কোডের কোন কিছুই অনুবাদ করা যাবে না। এখানে Hello World কে অনুবাদ করার প্রয়োজন নেই।

Reply:
> এখানে Hello World কে অনুবাদ করার প্রয়োজন নেই।
তা ঠিক। 

কিন্তু গাইডলাইন অনুযায়ী 
> কমেন্ট ব্যাতীত কোডের কোন কিছুই অনুবাদ করা যাবে না।
এটা ঠিক নয়। 

[https://github.com/javascript-tutorial/bn.javascript.info#text-in-code-blocks](https://github.com/javascript-tutorial/bn.javascript.info#text-in-code-blocks)

এখানে উদাহরণে "Hello, world" কে  'Hola mundo' তে ট্রান্সলেট করা হয়েছে। আমিও শুধু স্ট্রিঙগুলোই ট্রান্সলেট করেছি, কোন ভেরিয়েবল বা ক্লাসকে না। 

তাই আমার মনে হয় আপনারা চাইলে এধরণের অনুবাদ গ্রহণ করতে পারেন।

Reply:
Hello, world অনুবাদ করা যাবে। কোন সমস্যা নেই।

ধন্যবাদ আপনার মন্তব্যের জন্য! :smile: